web/templates/homepage/index.html.eex

```diff
@@ -53,12 +53,12 @@
   <% end %>
 </section>
 
-<section class="lm-bg-grey pa1 ph3 pb4 pb5-l" id="suggestion_form">
+<section class="pa4 tc-ns lm-bg-grey pa1 ph3 pb4 pb5-l" id="suggestion_form">
```

Comment:
Shouldn't the background for this be white? https://github.com/LDMW/app/issues/73#issuecomment-299250790